### ISSUE TITLE
ipdf - missing examples (Trac #1315)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1309]
add some

Migrated from https://code.icecube.wisc.edu/ticket/1315
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:14:55", 
    "description": "add some", 
    "reporter": "nega", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067295757382", 
    "component": "combo simulation", 
    "summary": "ipdf - missing examples", 
    "priority": "normal", 
    "keywords": "examples", 
    "time": "2015-08-30T03:37:51", 
    "milestone": "", 
    "owner": "kjmeagher", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150830T15:02:23: 
* olivas commented:

> Claudio is no longer the owner of ipdf.  It's orphaned an I need to find a new owner.


* olivas changed status from "new" to "assigned"
* olivas changed owner from "claudio.kopper" to "olivas"

**Comment by icecube-trac** [comment]
Trac update at 20151014T12:57:16: 
* kjmeagher changed owner from "olivas" to "kjmeagher"
* kjmeagher changed status from "assigned" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20160425T18:01:46: olivas commented:

> Please set a milestone.

**Comment by icecube-trac** [comment]
Trac update at 20160428T12:18:01: kjmeagher changed milestone from "" to "IceRec V5.1"

**Comment by icecube-trac** [comment]
Trac update at 20160506T12:13:18: 
* kjmeagher commented:

> In changeset 2568:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2568"
> redo the way icetray-inspect handles available libs, cmake writes a zero length file to `${I3_BUILD}/docs/no_inspect/` for projects which can't be inspected and icetray-inspect checks there for projects to skip
> fixes #1596,#1597 and #1309
> ```


* kjmeagher changed status from "accepted" to "closed"
* kjmeagher changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20160506T12:17:58: 
* kjmeagher commented:

> this was not fixed in r2568/IceTray I copied the wrong bug number


* kjmeagher changed status from "closed" to "reopened"
* kjmeagher changed resolution from "fixed" to ""

**Comment by icecube-trac** [comment]
Trac update at 20160726T10:04:29: 
* kjmeagher commented:

> In changeset 148751:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="148751"
> add example for ipdf python interface. fixes #1309
> ```


* kjmeagher changed status from "reopened" to "closed"
* kjmeagher changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190111T20:46:33: nwhitehorn commented:

> In changeset 168734:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="168734"
> redo the way icetray-inspect handles available libs, cmake writes a zero length file to `${I3_BUILD}/docs/no_inspect/` for projects which can't be inspected and icetray-inspect checks there for projects to skip
> fixes #1596,#1597 and #1309
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190112T00:17:55: nwhitehorn commented:

> In changeset 170271:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170271"
> redo the way icetray-inspect handles available libs, cmake writes a zero length file to `${I3_BUILD}/docs/no_inspect/` for projects which can't be inspected and icetray-inspect checks there for projects to skip
> fixes #1596,#1597 and #1309
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:14:55: 
* olivas commented:

> Milestone IceRec 5.2 deleted


* olivas changed milestone from "IceRec 5.2" to ""
